hudi-flink/src/main/java/org/apache/hudi/util/StreamerUtil.java

```diff
@@ -209,6 +209,8 @@ public static void initTableIfNotExists(Configuration conf) throws IOException {
           .setTableType(conf.getString(FlinkOptions.TABLE_TYPE))
           .setTableName(conf.getString(FlinkOptions.TABLE_NAME))
           .setPayloadClassName(conf.getString(FlinkOptions.PAYLOAD_CLASS))
+          .setPartitionColumns(conf.getString(FlinkOptions.PARTITION_PATH_FIELD, null))
```

Comment:
About this config option, we have specified the default value, right? Any reason, why we should set a new default value(`null`)?

Reply:
The default partition field name for flink is empty string but it is not work for spark partition prune. So here we return null if the partition field is not set.

Reply:
sounds good.